examples/react/public/index.html

```diff
@@ -2,24 +2,23 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="/logo192.png" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="/manifest.json" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```

Comment:
Did you test that this works?

Reply:
I was just changing this to match [the latest version of create-react-app](https://github.com/facebook/create-react-app/blob/f34d88e30c7d8be7181f728d1abc4fd8d5cd07d3/packages/cra-template/template/public/index.html#L17), but it just makes the existing 404 to the `manifest.json` worse. 

I'll just revert the changes in this file! Not sure if that dynamic [%PUBLIC%](https://create-react-app.dev/docs/using-the-public-folder/#docsNav) is webpack-specific. 

The only way I was able to prevent that 404 was to point specifically to the `/public/manifest.json` directory, but I'm guessing it needs to be more dynamic. 

And then even when pointed there, it was having issues parsing it's content, but changing the filename to `manifest.webmanifest` (https://developer.mozilla.org/en-US/docs/Web/Manifest) seemed to fix that for me.